content/_data/upgrades/2-361-1.adoc

```diff
@@ -0,0 +1,63 @@
+==== Require Java 11
```

Comment:
Since I originally wrote this text, I got some feedback on the bug tracker that it isn't as clear as it could be about the need to upgrade agents to Java 11 before upgrading the controller's JVM to Java 11 as well as the fact that builds can still be done with Java 8 as long as the agent Java process and controller Java process are running with Java 11. I will try and take some time tomorrow to revise the blog post to clarify these points, at which point you can use the same revised text here.

Reply:
Okay good to know, thank you again for sharing all this Basil, I'll be sure to add it to the upgrade guide once you've had a moment to do so. No worries/rush, and as always I appreciate your help and insight!